libs/accounts/src/lib/accounts-manager.spec.tsx

```diff
@@ -23,13 +23,23 @@ jest.mock('@vegaprotocol/react-helpers', () => ({
 describe('AccountManager', () => {
   it('change partyId should reload data provider', async () => {
     const { rerender } = render(
-      <AccountManager partyId="partyOne" onClickAsset={jest.fn} />
+      <AccountManager
+        partyId="partyOne"
+        onClickAsset={jest.fn}
+        isReadOnly={false}
```

Comment:
**chore (non-blocking)**: `isReadOnly` should probably get a test too. Non-blocking a it's such simple functionality